torch/cuda/amp/autocast_mode.py

```diff
@@ -6,7 +6,7 @@
     import numpy as np
     HAS_NUMPY = True
 except ModuleNotFoundError:
-    np = None
+    np = None  # type: ignore[assignment]
```

Comment:
Would `np : Any = None` work?

Not that it's important....

Reply:
Tried locally. mypy doesn't like that because `np` is already a `module` at that point.